mlrun/feature_store/retrieval/online.py

```diff
@@ -12,14 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import mlrun
 from mlrun.datastore.store_resources import ResourceCache
 from mlrun.datastore.targets import get_online_target
+from mlrun.feature_store.feature_vector import FixedWindowType
 from mlrun.serving.server import create_graph_server
 
 
+def _fixed_window_type_to_qbk_fixed_window_type(fixed_window_type):
+    try:
+        from storey import FixedWindowType as QueryByKeyFixedWindowType
+    except ImportError as exc:
+        raise ImportError(f"storey not installed, use pip install storey, {exc}")
+
+    if fixed_window_type == FixedWindowType.LastClosedWindow:
+        return QueryByKeyFixedWindowType.LastClosedWindow
+    if fixed_window_type == FixedWindowType.CurrentOpenWindow:
+        return QueryByKeyFixedWindowType.CurrentOpenWindow
+    return None
```

Comment:
Make this a method of the `FixedWindowType` enum itself, so you would do:
`FixedWindowType. CurrentOpenWindow.to_qbk_fixed_window_type()`